database/tbcd/level/level.go

```diff
@@ -1707,6 +1729,66 @@ func (l *ldb) BlockTxUpdate(ctx context.Context, direction int, txs map[tbcd.TxK
 	return nil
 }
 
+func (l *ldb) BlockKeystoneUpdate(ctx context.Context, direction int, keystones map[chainhash.Hash]tbcd.Keystone) error {
+	log.Tracef("BlockKeystoneUpdate")
+	defer log.Tracef("BlockKeystoneUpdate exit")
+
+	if !(direction == 1 || direction == -1) {
```

Comment:
can we define these directions as contants:
```
const DirectionForward = 1
const DirectionBackward = -1
```

or something similar? it's not clear what we're doing unless I read through this function (which may grow or become more complex over time)

Reply:
I agree with this. I believe we previously discussed having a Direction type but decided against it - this would significantly improve readability without adding a type.

Reply:
Yeah we keep having this conversation and while i do not expressively disagree in this case the code would overflow lines everywhere rendering it even more unreadable. And grade school math did tell us that negative is on the left and positive on the right.

I am sympathetic to the argument but code readability wins.